ShowPot/ShowPot/Common/LogHelper.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  LogHelper.swift
+//  ShowPot
+//
+//  Created by 이건준 on 6/12/24.
+//
+
+import Foundation
+import os.log
+
+enum LogHelper {
+    private static let subsystem = Bundle.main.bundleIdentifier!
```

Comment:
Force Unwrapping 지양해주세요

Reply:
```suggestion
    private static let subsystem = Bundle.main.bundleIdentifier ?? ""
```
흠... 이 부분은 강제언래핑을 사용하지않는다면 그냥 위 코드처럼 수정하는게 좋을까요 ?

Reply:
해당 부분은 문제가 있어도 빈 스트링으로 나머지 동작들을 실행하게 되어서, 예기치 못한 사이드 이슈가 많이 발생할 여지가 있어보여요. 
‼️`subsystem` **변수가 아무거나 들어가도 상관없는지 여부가 제일 중요할 것 같습니다**

Bundle ID가 조회되지 않는 상황은, 명백히 잘못된 상황으로 보여서 해당 구문을 중단시켜 주는 것이 맞을 것 같습니다. `fatalError`를 이용하는 것도 한 방법입니다
```suggestion
    private static let subsystem: String?
    
    init() {
        guard let bundleID = Bundle.main.bundleIdentifier else {
            return 또는 fatalError("Error: Cannot find Bundle ID")
        }
    }
```

개인적으로는, 초기화가 필요한 변수가 있기 때문에 `static` 보다는 싱글톤이 나을 수도 있다는 생각이 드네요